agent/provider/tests/manifest_negotiator.rs

```diff
@@ -455,20 +456,6 @@ fn manifest_negotiator_test_with_invalid_payload_signature(
     None;
     "Manifest accepted, because permissions are sufficient (has `unverified-permissions-chain` permission)"
 )]
-#[test_case(
-    Signature { private_key_file: Some("foo_req.key.pem"), algorithm: Some("sha256"), certificate: Some("foo_req.cert.pem")},
-    Some("NYI"), // cert_permissions_b64
-    &vec![CertPermissions::OutboundManifest],
-    Some("Audited-Payload rule: Not sufficient permissions. Required: `outbound-manifest|unverified-permissions-chain`, but has only: `outbound-manifest`"); // error msg
-    "Manifest rejected, because certificate has no `unverified-permissions-chain` permission."
-)]
-#[test_case(
-    Signature { private_key_file: Some("foo_req.key.pem"), algorithm: Some("sha256"), certificate: Some("foo_req.cert.pem")},
-    Some("NYI"), // cert_permissions_b64
-    &vec![CertPermissions::All],
-    Some("Audited-Payload rule: Not sufficient permissions. Required: `outbound-manifest|unverified-permissions-chain`, but has only: `all`"); // error msg
-    "Manifest rejected, even when permissions are set to `All` because `unverified-permissions-chain` permission is also required when Demand comes with its permissions"
-)]
```

Comment:
Why they are being deleted?

Reply:
`unverified-permissions-chain` is this "future proofing" permission which it is not used anywhere.

Reply:
Additionally I deleted from these tests every single case of passing `None` as a certificate permissions arriving in demand.
Cert permissions as a demand property were a late addition to this functionality, so I sprinkled them everywhere in tests, and now I just removed them, because it would be hard to figure out what these permissions are even for as they are not used anywhere during negotiation.